### PR TITLE
fix: pr fixes minor issues from previous openid4vp pr.

### PR DESCRIPTION
**Pull request by vkubiv**
Signed-off-by: Volodymyr Kubiv <volodymyr.kubiv@euristiq.com>

### CODE REVIEW

**Review by sudeshrshetty — APPROVED** [review]

File: cmd/wallet-sdk-gomobile/api/api.go [hunk 1]
```diff
@@ -26,8 +26,8 @@ type KeyReader interface {
 	// ExportPubKey returns the public key associated with the given keyID as raw bytes.
 	ExportPubKey(keyID string) ([]byte, error)
 
-	// GetSignAlgorithm returns sign algorithm name assisted with given key type.
-	GetSignAlgorithm(keyID string) (string, error)
+	// GetSigningAlgorithm returns sign algorithm name assisted with given key type.
```
Comment (DRK3) [inline cmd/wallet-sdk-gomobile/api/api.go:29]: Still says `sign` algorithm here

File: cmd/wallet-sdk-gomobile/api/api.go [hunk 2]
```diff
@@ -65,14 +65,13 @@ type CredentialWriter interface {
 }
 
 // Crypto defines useful Crypto operations.
-// TODO: Define more precisely the input and output formats.
 type Crypto interface {
-	// Sign will sign msg using a matching signature primitive in key referenced by keyID
+	// Sign will sign msg using a matching signature primitive from key referenced by keyID
```
Comment (DRK3) [inline cmd/wallet-sdk-gomobile/api/api.go:69]: I'm still not sure what this means... how do you get a signature primitive from a key?

Comment (vkubiv) [reply cmd/wallet-sdk-gomobile/api/api.go:69]: Crypto will get a signature primitive from keyID. Crypto and KMS implementation are always created as a pair. So Crypto implementation will know how to get signature primitive

File: cmd/wallet-sdk-gomobile/api/api.go (hunk 1 above)
Comment (DRK3) [reply cmd/wallet-sdk-gomobile/api/api.go:29]: Still says "assisted"

**Pull request merged**